README.md

```diff
@@ -43,6 +43,7 @@ Some general keyboard shortcuts that I use frequently.
 | Close Window | `shift-cmd-w` | `ctrl-shift-w`  | `ctrl-shift-w` | Closes the currently active editor window  |
 | Split Window | `cmd-k up/down/left/right` | `ctrl-k up/down/left/right` | `ctrl-k up/down/left/right` | Split the currently active tab in one of the four directions |
 | Focus Pane | `cmd-k cmd-up/down/left/right` | `ctrl-k ctrl-up/down/left/right` | `ctrl-k ctrl-up/down/left/right` | Move the focus to the pane in one of the four directions |
+| Toggle full screen | `F11` | `F11` | `F11` | Toggle full screen window
```

Comment:
On my Mac, pressing `F11` toggles the desktop view, i.e. moving all windows to the side so that the desktop can be seen. The keyboard shortcut for Full Screen Mode seems to be `ctrl-cmd-f` on the Mac, at least on mine...